library/src/amd_detail/hipsolver_sparse.cpp

```diff
@@ -1074,4 +1085,275 @@ catch(...)
     return hipsolver::exception2hip_status();
 }*/
 
+/******************** CSRLSVQR ********************/
+hipsolverStatus_t hipsolverSpScsrlsvqr(hipsolverSpHandle_t       handle,
+                                       int                       n,
+                                       int                       nnz,
+                                       const hipsparseMatDescr_t descrA,
+                                       const float*              csrVal,
+                                       const int*                csrRowPtr,
+                                       const int*                csrColInd,
+                                       const float*              b,
+                                       double                    tolerance,
+                                       int                       reorder,
+                                       float*                    x,
+                                       int*                      singularity)
+try
+{
+    if(!handle)
+        return HIPSOLVER_STATUS_NOT_INITIALIZED;
+    if(n < 0 || nnz < 0)
+        return HIPSOLVER_STATUS_INVALID_VALUE;
+    if(!descrA)
+        return HIPSOLVER_STATUS_INVALID_VALUE;
+    if(!csrRowPtr || !csrColInd || !csrVal || !descrA)
+        return HIPSOLVER_STATUS_INVALID_VALUE;
+    if(!b || !x || !singularity)
+        return HIPSOLVER_STATUS_INVALID_VALUE;
+
```

Comment:
I would suggest we keep the argument check on reorder, even though we're not using it at the moment. (Also applies to the other three functions)

Reply:
On second thought, I've decided to comment out the check on reorder. That way, we can say reorder is not referenced by the rocSOLVER backend, and having the comment there will remind us to reenable it once we using it.